lua/entities/starfall_processor/cl_init.lua

```diff
@@ -185,3 +185,75 @@ net.Receive("starfall_processor_used", function(len)
 	end
 end)
 
+local function getId( arg )
+	if not arg then
+		LocalPlayer():PrintMessage( HUD_PRINTCONSOLE, "Missing SteamID\n" )
+		return
+	end
+
+	-- Check if SteamID64
+	if tonumber( arg ) then
+		return util.SteamIDFrom64( arg ) or ""
+	-- Check if SteamID
+	--elseif string.match( arg, "^STEAM_%d:%d:%d+$" ) then
+	elseif string.StartsWith( arg, "STEAM_" ) then
+		-- TODO: Check if valid SteamID somehow
+		return arg
+	else
+		LocalPlayer():PrintMessage( HUD_PRINTCONSOLE, "Invalid SteamID\n" )
+		return
+	end
+end
+
+-- Local function to handle autocomplete for the terminate commands
+local function terminateAutoComplete( cmd, stringargs )
+	local examples = {}
+	for _, ply in pairs( player.GetHumans() ) do
+		--table.insert( examples, cmd .. " " .. "\"" .. ply:SteamID() .. "\"" )
+		table.insert( examples, cmd .. " " .. "\"" .. ply:SteamID() .. "\" " .. "\"" .. ply:Nick() .. "\"" )
+	end
+
+	return examples
+end
+
+-- TODO: documentation?
+---Terminates a user's starfall chips clientside
+---Supports SteamID and SteamID64
+concommand.Add( "sf_user_terminate_client", function( executor, cmd, args )
+	local id = getId( args[1] )
+	if not id or id == "" then return end
+
+	local ply = player.GetBySteamID( id )
+	if not ply or not ply:IsValid() then
+		LocalPlayer():PrintMessage( HUD_PRINTCONSOLE, "Player not found\n" )
+		return
+	end
+
+	for instance, _ in pairs( SF.playerInstances[ply] ) do
+		instance:Error( { message = "Killed by user", traceback = "" } )
+	end
+end, terminateAutoComplete, "Terminates a user's starfall chips clientside." )
+
+-- TODO: documentation?
+---Terminates a user's starfall chips
+---This command is admin only
+concommand.Add( "sf_user_terminate", function( executor, cmd, args )
+	if not executor:IsSuperAdmin() then return end
```

Comment:
this is supposed to be admin-only rather than superadmin-only.

Reply:
Yeah, A friend was shoulder-surfing and said it should be superadmin only. Will fix.